script.js

```diff
@@ -25,3 +26,186 @@ const Contact = document.querySelector('#Contact');
+  },
+  {
+    projectName: 'Tonic',
+    projectDescription:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    Technology: ['HTML', 'CSS', 'JavaScript'],
+    linkToLive: 'https://zafron047.github.io/Portfolio_Cohort-20_Mod-1/',
+    linkToSource: 'https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1',
+  },
+];
+
+const projectBtns = document.querySelectorAll('.see-project');
+Array.from(projectBtns).forEach((projectBtns, i) => {
+  projectBtns.addEventListener('click', () => {
+    const viewportWidth = window.innerWidth || document.documentElement.clientWidth;
+
+    if (viewportWidth >= 768) {
+      const popDiv = document.createElement('div');
+      popDiv.id = 'blur-background';
+      popDiv.classList.add('blur-background');
+      popDiv.style.position = 'fixed';
+      popDiv.style.display = 'block';
+      popDiv.style.top = '0';
+      popDiv.style.left = '0';
+      popDiv.style.width = '100vw';
+      popDiv.style.height = '100vh';
+      popDiv.style.zIndex = '11';
+      popDiv.style.backdropFilter = 'blur(6px)';
+      popDiv.innerHTML = `
+        <div class='d-card'>
+          <div class='D-Pop-Header'>
+            <div>
+              <h3 id='name'>${projectsArr[i].projectName}</h3>
+              <div id='exp' class='exp'>
+                <p>CANOPY</p>
+                <ul>
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ul>
+              </div>
+            </div>
+            <a><button id='desk-popup-close' class='popup-close'>X</button></a>
+          </div>
+
+          <img src='./assets/images/popup-img-desktop.svg'/>
+
+          <div class='card-details'>
+            <div class='card-details-left'>
+              <p class='project-info'>
+              ${projectsArr[i].projectDescription}
+              </p>
+            </div>
+
+            <div class='card-details-right'>
+              <ul class='skill'>
+                <li class='skill-box'>${projectsArr[i].Technology[0]}</li>
+                <li class='skill-box'>${projectsArr[i].Technology[1]}</li>
+                <li class='skill-box'>${projectsArr[i].Technology[2]}</li>
+              </ul>
+              <hr>
+              <div class='pop-btn'>
+                <a href='${projectsArr[i].linkToLive}'>
+                  <button class='d-pop-btn' href>
+                    See Live
+                    <i class='fa-solid fa-arrow-up-right-dots fa-2xs' style='color: #6070ff;'></i>
+                  </button>
+                </a>
+                <a href='${projectsArr[i].linkToSource}'>
+                  <button class='d-pop-btn'>
+                    See Source
```

Comment:
- [ ] I have to commend your efforsts on the button 👏 . However, kindly adjust the styling the have same paddings and inner margins as that on the figma design.
- You'll agree with me that this gives a better user experince that yours 👍 

## Your view:
![image](https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1/assets/50990925/4abfd906-432c-4dee-9bb7-34067e5f8219)

## Figma::
![image](https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1/assets/50990925/1257c718-1206-480a-9a00-028d0517d590)

Reply:
I have given the button a responsive height increase.

![Screenshot 2023-05-20 053547](https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1/assets/117600135/2be8e96a-e559-42e3-8234-dc00c66e3a90)